wordpress-mobile/WordPress-iOS
# Readability: Revised People Management layout.

This removes the previously added toggle for forcing fixed width cells via `WPTableViewCell` and revises the layout to follow readable margins instead.

- Also drops the constant values used for trailing and leading (no longer needed)
- Also changed the name label font to use the system font instead of Merriweather.

To test:

1. Ensure the People management views follow readable margins on iPad.
2. Ensure the same People management views follow the expected margins on iPhone devices.

Note: In the future, this layout could probably use some `UIStackView` goodness.

Needs review: @frosty can you take a look? 🕶
